### PR TITLE
Go strings Fields() added #2563

### PR DESCRIPTION
**Pull request by sumitdethe27**
### Type of Change

- Adding a new entry

### Checklist

- [ x] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/strings/terms/fields/fields.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+```
+
+The output will be:
+
+```shell
+[Hello World Golang]
+```
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/go/concepts/strings/terms/fields/fields.md:49]: The codebyte is exactly the same as the example above. Can we make it different somehow?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@sumitdethe27 Looks good overall, but we'd like to see a codebyte that is different from the first example given.

File: content/go/concepts/strings/terms/fields/fields.md (hunk 1 above)
Comment (sumitdethe27) [reply content/go/concepts/strings/terms/fields/fields.md:49]: Sure.

Comment (SSwiniarski) [reply content/go/concepts/strings/terms/fields/fields.md:49]: @sumitdethe27 I meant doing something different than using the function and just displaying the result. Maybe loop through the `fields` variable and print it that way?

Comment (sumitdethe27) [reply content/go/concepts/strings/terms/fields/fields.md:49]: Oh!, Okay i got it.

**Comment by sumitdethe27** [comment]
is this version okay? @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@sumitdethe27 Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@sumitdethe27 @SSwiniarski Just made a couple minor edits. Looks good to merge

**Comment by sumitdethe27** [comment]
@caupolicandiaz @SSwiniarski Thankyou for reviewing.

**Comment by SSwiniarski** [comment]
@sumitdethe27 Congrats on the new entry! Live link here:https://www.codecademy.com/resources/docs/go/strings/fields

image

**Pull request merged**